ontology/yaml/resources/ELECTRICAL/entity_types/ABSTRACT.yaml

```diff
@@ -63,6 +63,21 @@ IOBM:
   - load_percentage_sensor
   - bypass_phase_status
 
+OM:
+  description: "output electrical monitoring, typically for a UPS."
+  is_abstract: true
+  uses:
+  - output_phase1_neutral_voltage_sensor
+  - output_phase2_neutral_voltage_sensor
+  - output_phase3_neutral_voltage_sensor
+  - output_phase1_current_sensor
+  - output_phase2_current_sensor
+  - output_phase3_current_sensor
+  - output_frequency_sensor
+  - output_power_sensor
+  - output_apparent_power_sensor
+  - bypass_status
```

Comment:
Suggest making this field optional, see optional fields on `IOBM`

Reply:
Please move frequency, power, and apparent power fields to optional as well to match `IOBM`.